src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -86,12 +86,12 @@
       "choices": [
         {
           "choice": "net6.0",
-          "displayName": ".NET 6.0 (Long Term Support)",
+          "displayName": ".NET 6.0",
           "description": "Target .NET 6.0 (Long Term Support)"
         },
         {
           "choice": "net7.0",
-          "displayName": ".NET 7.0 (Standard Term Support)",
```

Comment:
These values were specifically chosen to mimic the values you would get from an Official Microsoft template
![image](https://user-images.githubusercontent.com/3860573/220618719-90d4166a-a59e-4b2f-8d59-86719b5fadfc.png)

Reply:
We're not using that UI, so irrelevant - it's still included in the description for that reason